### PR TITLE
CLOUD-209-sync-options-updated-operator-topic

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated and restructured operator.md.

### CODE REVIEW

File: configure/operator.md [hunk 1]
```diff
@@ -17,26 +17,31 @@ The ``pxc`` section in the deploy/cr.yaml file contains general configuration op
...
 |resources.limits.cpu            | string     | `1`       | [Kubernetes CPU limit](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container) for a PXC container |
+|nodeSelector                    | label      | `disktype: ssd`        | The [Kubernetes nodeSelector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector) constraint|
+|affinity.topologyKey            | string     |`kubernetes.io/hostname`| The [Operator topologyKey](./constraints) node anti-affinity constraint|
+|affinity.advanced               | subdoc     |           | If available, it makes [topologyKey](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature) node affinity constraint to be ignored |
+|affinity.tolerations                     | subdoc     | `node.alpha.kubernetes.io/unreachable` | The [Kubernetes Pod tolerations] (https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)            |
```
Comment (fiowro) [inline configure/operator.md]:
```suggestion
|tolerations                     | subdoc     | `node.alpha.kubernetes.io/unreachable` | The [Kubernetes Pod tolerations] (https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)            |
```

File: configure/operator.md [hunk 2]
```diff
@@ -17,26 +17,31 @@ The ``pxc`` section in the deploy/cr.yaml file contains general configuration op
 | Key                            | Value Type | Example   | Description |
 |--------------------------------|------------|-----------|-------------|
 |size                            | int        | `3`       |  The size of the Percona XtraDB Cluster, must be >= 3 for [High-Availability](hhttps://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html) |
-|image                           | string     |`perconalab/pxc-openshift:0.1.0` | Percona XtraDB Cluster docker image to use                                                                     |
+| allowUnsafeConfigurations      | string     | `false`   | Prevents users from configuring a cluster with unsafe parameters |
+|image                           | string     |`perconalab/percona-xtradb-cluster-operator:0.4.0-pxc` | Percona XtraDB Cluster docker image to use                                                                     |
```
Comment (delgod) [inline configure/operator.md]:
```suggestion
|image                           | string     |`percona/percona-xtradb-cluster-operator:1.0.0-pxc` | Percona XtraDB Cluster docker image to use                                                                     |
```

File: configure/operator.md [hunk 3]
```diff
@@ -46,25 +51,35 @@ The ``proxysql`` section in the deploy/cr.yaml file contains configuration optio
 |--------------------------------|------------|-----------|-------------|
 |enabled                         | boolean    | `true`    | Enables or disables [load balancing with ProxySQL](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html) [Service](https://kubernetes.io/docs/concepts/services-networking/service/) |
 |size                            | int        | `1`       | The number of the ProxySQL daemons [to provide load balancing](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html), must be = 1 in current release|
-|image                           | string     |`perconalab/proxysql-openshift:0.1.0` | ProxySQL docker image to use |
+|image                           | string     |`pperconalab/percona-xtradb-cluster-operator:0.4.0-proxysql` | ProxySQL docker image to use |
```
Comment (delgod) [inline configure/operator.md]:
```suggestion
|image                           | string     |`percona/percona-xtradb-cluster-operator:1.0.0-proxysql` | ProxySQL docker image to use |
```

File: configure/operator.md [hunk 4]
```diff
@@ -83,16 +98,18 @@ The ``backup`` section in the [deploy/cr.yaml](https://github.com/percona/percon
 
 | Key                            | Value Type | Example   | Description |
 |--------------------------------|------------|-----------|-------------|
-|image                           | string     | `perconalab/backupjob-openshift:0.2.0` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
+|image                           | string     | `perconalab/percona-xtradb-cluster-operator:0.4.0-backup` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
```
Comment (delgod) [inline configure/operator.md]:
```suggestion
|image                           | string     | `percona/percona-xtradb-cluster-operator:1.0.0-backup` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
```

File: configure/operator.md [hunk 5]
```diff
@@ -83,16 +98,18 @@ The ``backup`` section in the [deploy/cr.yaml](https://github.com/percona/percon
...
 |storages.s3.credentialsSecret   | string     | `my-cluster-name-backup-s3`| [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) for backups. It should contain `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` keys. |
 |storages.s3.bucket              | string     |           | The [Amazon S3 bucket](https://docs.aws.amazon.com/en_us/AmazonS3/latest/dev/UsingBucket.html) name for backups     |
 |storages.s3.region              | string     |`us-east-1`| The [AWS region](https://docs.aws.amazon.com/en_us/general/latest/gr/rande.html) to use. Please note **this option is mandatory** not only for Amazon S3, but for all S3-compatible storages.|
-|storages.s3.endpointUrl         | string     |           | The endpoint URL of the S3-compatible storage to be used (not needed for the original Amazon S3 cloud)   |
+|storage.s3.endpointUrl | string|           | The endpoint URL of the S3-compatible storage to be used (not needed for the original Amazon S3 cloud) |
+|storage.type                    | string    | `filesystem` | persistent volume type |
```
Comment (delgod) [inline configure/operator.md]: duplicate, see line 88, 103

File: configure/operator.md [hunk 6]
```diff
@@ -83,16 +98,18 @@ The ``backup`` section in the [deploy/cr.yaml](https://github.com/percona/percon
...
 |storages.persistentVolumeClaim.storageClassName | string | `standard`| Set the [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) to use with the PXC backups [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) for the `filesystem` storage type                    |
 |storages.persistentVolumeClaim.accessModes | array | ["ReadWriteOnce"] | The [Kubernetes Persistent Volume access modes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) |
+|storage | string | `6Gi`| the storage size |
```
Comment (delgod) [inline configure/operator.md]: what option is it?

File: configure/operator.md [hunk 7]
```diff
@@ -46,25 +51,35 @@ The ``proxysql`` section in the deploy/cr.yaml file contains configuration optio
...
+| podDisruptionBudget.maxUnavailable   | int  | `1`    | [Kubernetes Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) The number of pods unavailable after eviction| 
+| podDisruptionBudet.minAvailable      | int  | `0`   | [Kubernetes Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) The number of pods available after eviction |
+
+
+
+
+
+
+
+
```
Comment (delgod) [inline configure/operator.md]: why so many new lines?

File: configure/operator.md (hunk 4 above)
Comment (patrickbirch) [reply configure/operator.md]: Updated in current version.

File: configure/operator.md (hunk 5 above)
Comment (patrickbirch) [reply configure/operator.md]: One is for the persistentvolume - added text to make the distinction.

File: configure/operator.md (hunk 6 above)
Comment (patrickbirch) [reply configure/operator.md]: persistentvolume storage - added text to make the distinction.

File: configure/operator.md (hunk 7 above)
Comment (patrickbirch) [reply configure/operator.md]: Deleted empty lines.

**Comment by patrickbirch** [comment]
Outdated

**Comment by patrickbirch** [comment]
outdated

**Pull request closed without merging**